Gems/ROS2PoseControl/Code/Source/Clients/ROS2PoseControl.h

```diff
@@ -82,5 +85,9 @@ namespace ROS2PoseControl
         std::shared_ptr<tf2_ros::TransformListener> m_tf_listener{ nullptr };
         std::unique_ptr<tf2_ros::Buffer> m_tf_buffer;
 
+        bool m_tf2WarningShown = false;
+        bool m_groundNotFoundWarningShown = false;
+        bool m_startingOffsetNotFoundWarningShown = false;
```

Comment:
These 3 values are newly defined and seem not to be used anywhere.

Reply:
This is comming from this commit 6270c65867e5483d77039a1e82e900150ded0ab4, which I though was merged